src/reducers/settings.js

```diff
@@ -18,6 +18,7 @@ const defaultState: SettingsState = {
     isEnabled: false,
     value: '',
   },
+  currencies: [],
```

Comment:
you might want to store them by coinId in future for performance.

Reply:
That's right